src/crypto/CMakeLists.txt

```diff
@@ -48,8 +48,11 @@ set(crypto_sources
   random.c
   skein.c
   slow-hash.c
+  rx-slow-hash.c
```

Comment:
Push this line up 2 lines to keep consistent alphabetical order.

Reply:
It dont need to be in alphabetical order

Reply:
![97571858675ffa631283be27f1d4ef6160fe9053](https://user-images.githubusercontent.com/29845135/66810803-44029780-ef30-11e9-9922-7aa2100f4dd0.jpeg)